generic_config_updater/generic_updater.py

```diff
@@ -122,6 +125,14 @@ def replace(self, target_config):
 
         self.logger.log_notice("Config replacement completed.")
 
+    @staticmethod
+    def switchport_mode_remove(config):
```

Comment:
Please add some comments for the background reason for removing the config['PORT'][port]['mode']

Reply:
Added as per suggestion, thanks